lib/collection/src/collection.rs

```diff
@@ -1156,6 +1170,11 @@ impl Collection {
                             .unwrap_or(new_diff)
                     });
                 }
+
+                // Update vector quantization config
+                if let Some(ref config) = update_params.quantization_config {
+                    vector_params.quantization_config.replace(config.clone());
+                }
```

Comment:
Potential problem: it is currently impossible to unset (remove) quantization. It can only be updated with a new setting.

Reply:
In a call with Andrey (a while back) we agreed that we accept this limitation.

Reply:
Could someone, please, clarify this a bit? What's the issue with `null`s in Python client, and how this issue with Python client prevents us from allowing to disable quantization with explicit `null`? Like, I get that it might be impossible to disable quantization while using Python client, but at least it would be possible with REST and/or in other clients?